Resources/Private/Forms/Contact.form.yaml

```diff
@@ -8,13 +8,10 @@ finishers:
         identifier: EmailToReceiver
         options:
             subject: '{subject}'
-            recipientAddress: ''
-            recipientName: ''
             senderAddress: '{email}'
             senderName: '{fullname}'
```

Comment:
These must also be migrated.

Reply:
According the sources this looks still fine https://github.com/TYPO3/typo3/blob/10.4/typo3/sysext/form/Classes/Domain/Finishers/EmailFinisher.php#L54-L55

Reply:
Yes, got confused myself. This of course is still a single value.

Reply:
So these must be left unchanged.